.changeset/forty-oranges-joke.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/integration-react': minor
```

Comment:
Think that this should just be a `patch` release right? `integration-react` is `0.x` package, and this change isn't breaking as far as I can tell 🤞

Reply:
https://github.com/backstage/backstage/pull/23175#discussion_r1499231251

I originally had it as a patch, but changed based on feedback.

Let me know which one is correct, and I'll change it.

Reply:
100% my fault, clearly my brain was not working at the time. Sorry about that @Phiph

Reply:
No worries :) I'll make the change now :)